lib/ansible/plugins/action/yum.py

```diff
@@ -47,7 +47,22 @@ def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
         # Carry-over concept from the package action plugin
-        module = self._task.args.get('use_backend', "auto")
```

Comment:
This one-liner will also work:
```python3
module = self._task.args.get('use_backend', self._task.args.get('use', 'auto'))
```
But it does not through a proper error when both **use_backend** and **use** is used.
Let me know.